ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -7,6 +7,194 @@
   },
   timelineFile: 'e5s.txt',
   triggers: [
+    {
+      id: 'E5S Fury\'s Bolt Gain',
+      regex: Regexes.gainsEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
+      regexDe: Regexes.gainsEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
```

Comment:
Sorry for one last comment, but can you remove translations for regexes that are not correct? I think it makes it much harder for translators to know if there's a missing translation or not.

Reply:
I made mistake there. Ramuh was the same for en, de, fr. I don't know how exactly I was thinking, but that was the problem.